client/cache_test.go

```diff
@@ -0,0 +1,145 @@
+// Copyright Contributors to the Open Cluster Management project
+package client
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/discovery"
+)
+
+var _ = Describe("Test the cache", Ordered, func() {
```

Comment:
Ordered need? looks like `It` is independent.  we cannot test with parallel though

Reply:
Ginkgo didn't allow me to execute the test if it wasn't ordered due to the `BeforeEach`.

Reply:
I am also surprised that the `Ordered` is required.